packages/babel-plugin-emotion/src/utils/label.js

```diff
@@ -65,6 +66,11 @@ function getDeclaratorName(path, t) {
     return ''
   }
 
+  // we could also have an object property
+  if (parent.isObjectProperty()) {
+    return parent.node.key.name || parent.node.key.value
```

Comment:
This suggests that there should be 2 tests added - one for each case.

Reply:
The fallback is purely speculative based on related code inside `getIdentifierName` -- I'll just remove it.